docs/source/app_developers_guide/aws.rst

```diff
@@ -0,0 +1,427 @@
+  [20:52:07.834 [MainThread] handler DEBUG] Incrementing "ARqIDG" by 6
+
+
+Submitting Transactions With Sawtooth Submit
+++++++++++++++++++++++++++++++++++++++++++++
+
+You can also submit transactions, including intkey transactions, with the 
+``sawtooth batch submit`` command.
+
+For example, submit the transactions in the file ``batches.intkey`` generated above
+with this command:
+
+.. code-block:: console
+
+  $ sawtooth batch submit -f batches.intkey
+
+
+Viewing the Block Chain
+-----------------------
+
+You can view the blocks stored in the blockchain using the sawtooth CLI.
+
+.. note::
+
+  The sawtooth CLI provides help for all subcommands. For example, to get help
+  for the ``block`` subcommand, enter the command ``sawtooth block -h``.
+
+Viewing List Of Blocks
+++++++++++++++++++++++
+
+Enter the command ``sawtooth block list`` to view the blocks stored by the state:
+
+.. code-block:: console
+
+  $ sawtooth block list
+
+The output of the command will be similar to this:
+
+.. code-block:: console
+
+  NUM  BLOCK_ID
+  8    22e79778855768ea380537fb13ad210b84ca5dd1cdd555db7792a9d029113b0a183d5d71cc5558e04d10a9a9d49031de6e86d6a7ddb25325392d15bb7ccfd5b7  2     8     02a0e049...
+  7    c84346f5e18c6ce29f1b3e6e31534da7cd538533457768f86a267053ddf73c4f1139c9055be283dfe085c94557de24726191eee9996d4192d21fa6acb0b29152  2     20    02a0e049...
+  6    efc0d6175b6329ac5d0814546190976bc6c4e18bd0630824c91e9826f93c7735371f4565a8e84c706737d360873fac383ab1cf289f9bf640b92c570cb1ba1875  2     27    02a0e049...
+  5    840c0ef13023f93e853a4555e5b46e761fc822d4e2d9131581fdabe5cb85f13e2fb45a0afd5f5529fbde5216d22a88dddec4b29eeca5ac7a7b1b1813fcc1399a  2     16    02a0e049...
+  4    4d6e0467431a409185e102301b8bdcbdb9a2b177de99ae139315d9b0fe5e27aa3bd43bda6b168f3ac8f45e84b069292ddc38ec6a1848df16f92cd35c5bd6e6c9  2     20    02a0e049...
+  3    9743e39eadf20e922e242f607d847445aba18dacdf03170bf71e427046a605744c84d9cb7d440d257c21d11e4da47e535ba7525afcbbc037da226db48a18f4a8  2     22    02a0e049...
+  2    6d7e641232649da9b3c23413a31db09ebec7c66f8207a39c6dfcb21392b033163500d367f8592b476e0b9c1e621d6c14e8c0546a7377d9093fb860a00c1ce2d3  2     38    02a0e049...
+  1    7252a5ab3440ee332aef5830b132cf9dc3883180fb086b2a50f62bf7c6c8ff08311b8009da3b3f6e38d3cfac1b3ac4cfd9a864d6a053c8b27df63d1c730469b3  2     120   02a0e049...
+  0    8821a997796f3e38a28dbb8e418ed5cbdd60b8a2e013edd20bca7ebf9a58f1302740374d98db76137e48b41dc404deda40ca4d2303a349133991513d0fec4074  0     0     02a0e049...
+
+
+Viewing A Particular Block
+++++++++++++++++++++++++++
+
+From the output generated by the ``sawtooth block list`` command, copy the id
+of a block you want to get more info about, then paste it where specified below
+in the ``sawtooth block show`` command:
+
+.. code-block:: console
+
+  $ sawtooth block show {BLOCK ID}
+
+The output of this command includes all data stored under that block, and can be
+quite lengthy. Is should look something like this:
```

Comment:
s/Is/It/.  (Sorry - I should have caught this in my first pass.)

Reply:
This was copied from existing documentation. I suggest this go in a later docs cleanup PR.